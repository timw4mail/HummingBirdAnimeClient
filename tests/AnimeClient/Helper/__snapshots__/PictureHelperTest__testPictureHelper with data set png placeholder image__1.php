<?php declare(strict_types=1);

return '<img src="https://localhost/assets/images/placeholder.png" alt="placeholder.png" loading="lazy" />';
