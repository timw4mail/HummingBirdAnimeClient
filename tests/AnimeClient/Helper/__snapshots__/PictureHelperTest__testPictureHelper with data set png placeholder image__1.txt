<img src="https://localhost/assets/images/placeholder.png" alt="placeholder.png" loading="lazy" />